src/journal-remote/fuzz-journal-remote.c

```diff
@@ -35,13 +34,10 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         assert_se(mkdtemp_malloc("/tmp/fuzz-journal-remote-XXXXXX", &tmp) >= 0);
         assert_se(name = path_join(tmp, "fuzz-journal-remote.XXXXXX.journal"));
 
-        fdin = fdin_close = memfd_new_and_map("fuzz-journal-remote", size, &mem);
+        fdin = fdin_close = memfd_new_and_seal("fuzz-journal-remote", data, size);
```

Comment:
Looks like `journal-remote` isn't fuzzed on OSS-Fuzz at all: https://storage.googleapis.com/oss-fuzz-coverage/systemd/reports/20241218/linux/src/systemd/src/journal-remote/report.html. I'm not sure if it's related to this PR though.

Reply:
I reported it in https://github.com/google/oss-fuzz/issues/12519#issuecomment-2558144965

Reply:
That being said it crashes a lot and it was bisected to https://github.com/systemd/systemd/pull/32592 apparently so it could be those crashes are related as well. Though I'm not sure why it all started failing in November.

Reply:
@evverx Please open a new issue, rather than commenting on already closed PR. Otherwise, we will easily forget your comment.

Reply:
@yuwata there is nothing to report here. The coverage went down most likely because of an OSS-Fuzz issue (and I reported it there. There seem quite a few projects where it happened in November).

The fuzz-journal-remote crash was reported in https://oss-fuzz.com/testcase-detail/4645490914820096.

Reply:
The coverage report is fixed: https://storage.googleapis.com/oss-fuzz-coverage/systemd/reports/20241223/linux/src/systemd/src/journal-remote/report.html

To judge from the ClusterFuzz commits where it was fixed it was caused by the fuzz-journal-remote crashes (if it hadn't crashed the coverage report would have been OK). Either way it was an OSS-Fuzz regression.

The crash has just been mirrored on GitHub (https://github.com/systemd/systemd/issues/35737) to make it a bit more visible. I'm not sure why it wasn't mirrored. It has been crashing since July as far as I can see.